### DISCUSSION TITLE
Hey, hope all is well! Just curious did anyone have trouble on Solidity Smart Contract Development 14. Zksync Plugin

### DISCUSSION THREAD

**Discussion by stevenmarquez-git** [Cyfrin/foundry-full-course-cu#2507] [Q&A]
My problem is that I'm able to compile the SimpleStorage.sol in Zksync, but it gives an error when I go to deploy and verify. After I click confirm through MetaMask, it doesn't deploy and displays the message "Contract SimpleStorage failed to deploy!" when I hover over the Zksync icon. 

-Wallet is Connected.
-Zksync Sepolia Testnet is Connected.
-Followed Lesson 15. Zksync Plugin instructions just in case.

What do you think could be the issue?

**Comment by EngrPips** [comment]
Hello @stevenmarquez-git, Please try without `verify` and let's if it works.
